gatsby_fp/src/components/fieldComponents/genInfoTable.js

```diff
@@ -0,0 +1,87 @@
+import React from "react"
+
+export default function GenInfoTable
+({school, title, imgsrc, building, room, phone, fax, username, website}){
+
+  return(
+    <>
+    <div className="table_wrapper">
+          <div className="table_wrapper_inner">
+            <table
+              cellPadding="5"
+              cellSpacing="0"
+              border="0"
+              className="bioregion"
+              width="100%"
+            >
+              <tbody>
+                <tr>
+                  <td valign="top" align="left" width="160px">
+                    <img
+                      src={`data:image/jpeg;base64,${imgsrc}`}
+                      className="bioimage"
+                      style={{ maxWidth: "150px" }}
+                      alt=""
+                    />
+                  </td>
+                  <td valign="top" align="left">
```

Comment:
Same thing here. The existing markup probably has this, but now that we have the chance to change it, we should move this into our CSS, using classes to select and style it.